src/components/App/index.tsx

```diff
@@ -1,32 +1,55 @@
-import React from 'react';
+import React, { useState } from 'react';
 
 import {
   HappiGraph,
   HappiGraphActions
 } from '../HappiGraph';
 
+import { GraphType } from "../HappiGraph/happi-graph.helpers";
+
 import './index.scss';
 import '../HappiGraph/happi-graph.scss';
 
 import { mockData } from '../../mockData';
+import { Modal } from '@mantine/core';
 
 const rawData = {
   ...mockData
 };
 
 
 export function App() {
+  const [selectedNodeData, setSelectedNodeData] = useState(undefined);
+  const [opened, setOpened] = useState(false);
+
   return <>
     <div className="container">
-      <HappiGraph rawData={{...rawData}}
-                  algorithm={""}
-                  debug={false}
-                  printMode={false}
-                  graphDirection={"VERTICAL"}
-                  selectedNodeId={"term@68e36496-7167-4af7-abdd-a0cd36e24084:6662c0f2.e1b1ec6c.66k78i6du.uchsna1.rn2epa.rfn2fjqf7h4qvmt5lflm8"}
-                  actions={<HappiGraphActions rawData={{...rawData}}/>}
-                  onNodeClick={(d: any) => console.log(d)}
-                  onGraphRender={() => { console.log('Graph rendered');}} />
+      <div style={{textAlign: 'center'}}>
+        <h1>TEx</h1>
+      </div>
+
+      <Modal
```

Comment:
extract tex functionality in separate demo component so that we can differentiate and encapsulate the functionality

ideally the modal should be implemented in the egeria-ui-components where there will be a new happi-graph instance

Reply:
I've changed the frontend to have a dropdown menu to choose the graph type, the only issue I'm struggling with a bit is to have the Happi Graph update with the new Graph Type state after you change it with the dropdown. If you run the code it should make sense. For example, whatever graph type you start with is ok, but then if you change the graph via the down from Inheritance to Lineage, the graph does not refresh to show the Lineage graph.